### ISSUE TITLE
ihaskell-0.6.4.0 fails to build in stackage nightly

### ISSUE THREAD

**Issue by juhp** [gibiansky/IHaskell#547]
ihaskell doesn't seem to build in the Stackage Nightly build:

```
[19 of 22] Compiling IHaskell.Eval.Evaluate ( src/IHaskell/Eval/Evaluate.hs, dist/build/IHaskell/Eval/Evaluate.o )

src/IHaskell/Eval/Evaluate.hs:195:66:
    Couldn't match expected type ‘[Char]’
                with actual type ‘Maybe String’
    In the second argument of ‘isPrefixOf’, namely ‘idString’
    In the first argument of ‘guard’, namely
      ‘(iHaskellPkgName `isPrefixOf` idString)’

src/IHaskell/Eval/Evaluate.hs:202:63:
    Couldn't match type ‘Maybe String’ with ‘[Char]’
    Expected type: [[Char]]
      Actual type: [Maybe String]
    In the second argument of ‘filter’, namely ‘packageNames’
    In the expression: filter (== iHaskellPkgName) packageNames

src/IHaskell/Eval/Evaluate.hs:210:88:
    Couldn't match expected type ‘[Char]’
                with actual type ‘Maybe String’
    In the second argument of ‘stripPrefix’, namely ‘pkgName’
    In the expression: stripPrefix initStr pkgName

src/IHaskell/Eval/Evaluate.hs:211:71:
    Couldn't match type ‘[Char]’ with ‘Maybe String’
    Expected type: [Maybe String]
      Actual type: [String]
    In the second argument of ‘notElem’, namely ‘broken’
    In the expression: pkgName `notElem` broken

src/IHaskell/Eval/Evaluate.hs:228:41:
    Couldn't match type ‘Maybe String’ with ‘[Char]’
    Expected type: [String]
      Actual type: [Maybe String]
    In the second argument of ‘map’, namely ‘displayPackages’
    In the expression: map toImportStmt displayPackages
```

**Comment by gibiansky** [comment]
This is 7.10.2, yeah?

**Comment by juhp** [comment]
Right, ghc-7.10.2

**Comment by gibiansky** [comment]
fixed with 0.6.4.1, on hackage now
